manifest.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: powerdummy-populator
+  namespace: relationships
+  labels:
+    app: powerdummy
```

Comment:
be consistent with the app label. powerdummy-populator or powerdummy ?

Reply:
fixed